### PR TITLE
Update voice and shard examples

### PR DESCRIPTION
**Pull request by PlayTheFallen**
**Please describe the changes this PR makes and why it should be merged:**
Update examples so that users are not mistaken for old versions of the library.
Extended PR of #1382

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: test/shard.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ client.on('message', msg => {
   if (msg.content.startsWith('?eval') && msg.author.id === '66564597481480192') {
     try {
       const com = eval(msg.content.split(' ').slice(1).join(' '));
-      msg.channel.sendMessage('```\n' + com + '```');
+      msg.channel.send('```\n' + com + '```');
```
Comment (bdistin) [inline test/shard.js]: `msg.channel.send(com, { code: true });`

File: test/shard.js [hunk 2]
```diff
@@ -10,9 +10,9 @@ client.on('message', msg => {
...
     } catch (e) {
-      msg.channel.sendMessage('```\n' + e + '```');
+      msg.channel.send('```\n' + e + '```');
```
Comment (bdistin) [inline test/shard.js]: `msg.channel.send(e, { code: true });`

File: test/voice.js [hunk 3]
```diff
@@ -51,10 +51,10 @@ client.on('message', m => {
   } else if (m.content.startsWith('#eval') && m.author.id === '66564597481480192') {
     try {
       const com = eval(m.content.split(' ').slice(1).join(' '));
-      m.channel.sendMessage(`\`\`\`\n${com}\`\`\``);
+      m.channel.send(`\`\`\`\n${com}\`\`\``);
```
Comment (bdistin) [inline test/voice.js]: `msg.channel.send(com, { code: true });`

File: test/voice.js [hunk 4]
```diff
@@ -51,10 +51,10 @@ client.on('message', m => {
...
     } catch (e) {
       console.log(e);
-      m.channel.sendMessage(`\`\`\`\n${e}\`\`\``);
+      m.channel.send(`\`\`\`\n${e}\`\`\``);
```
Comment (bdistin) [inline test/voice.js]: `msg.channel.send(e, { code: true });`

**Comment by PlayTheFallen** [comment]
@bdistin updated format.
Waiting on travis to approve

File: test/voice.js [hunk 5]
```diff
@@ -51,10 +51,10 @@ client.on('message', m => {
   } else if (m.content.startsWith('#eval') && m.author.id === '66564597481480192') {
     try {
       const com = eval(m.content.split(' ').slice(1).join(' '));
-      m.channel.sendMessage(`\`\`\`\n${com}\`\`\``);
+      m.channel.send(com, { code: true }\`\`\``);
```
Comment (bdistin) [inline test/voice.js]: "\`\`\``" is going to cause issues

Comment (PlayTheFallen) [inline test/voice.js]: oops

**Pull request merged**